internal/client/consensus/grandpa/authorities.go

```diff
@@ -8,6 +8,9 @@ import (
 	"fmt"
 	"sync"
 
+	"github.com/ChainSafe/gossamer/internal/client/telemetry"
+	pgrandpa "github.com/ChainSafe/gossamer/internal/primitives/consensus/grandpa"
```

Comment:
nit/maybe rename to grandpa and finality-grandpa, pgrandpa feels unintuitive to me